test/common_utils.py

```diff
@@ -16,6 +16,17 @@
 
 
 def get_bool_env_var(name, *, exist_ok=False, default=False):
+    """Gets the value of a boolean environment variable, by mapping common bool-ish
+    values to a :class:`bool`.
+
+    Args:
+        name: Name of the environment variable.
+        exist_ok: Returns ``True`` if the environment variable exists regardless of the
+            value it holds. This is useful for checking if an environment variable
+            exists even if it holds no value.
```

Comment:
Thanks for adding the docstring.


If a variable has no value, I believe it will be evaluated to `None` and thus the default will be returned instead. Perhaps you mean an empty string?

```py
➜  ~ export HELLO && python -c "import os; print(os.getenv('HELLO'))"
None
➜  ~ export HELLO= && python -c "import os; print(os.getenv('HELLO'))"
<empty_string>
```

-----

This is for another PR, but I believe we should not use this helper, nor `run_on_env_var()`. We are effectively converting simple calls to `@pytest.skipif(os.getenv(...) == ...)` into a deeper call stack, all the way down to `distutils.strtobool()`. Also, the fact that `exist_ok` is only ever used once in our code base is a sign that this might not be the abstraction level that we're looking for.

Reply:
Before I added `get_bool_env_var` we had this

https://github.com/pytorch/vision/blob/cca16993c923d96592f27e9a7d8f416889c602fe/test/common_utils.py#L16-L18

So when adding a flag from an environment variable, one needs to know exactly which true value needs to be used. With the function this goes away in the sense that we accept all values as long as they are true-ish.

Furthermore `run_on_env_var` communicates the intent of the decorator better than `skipif`. This test is not skipped because of some missing dependency or what ever, but the user has to actively do something to run the test.

Reply:
I'm not sure I agree (or perhaps understand) that last statement, but I won't insist.

Going back to my first comment, we might need to clarify the "even if it holds no value" part.

Reply:
See #5222.